### PR TITLE
support more properties in create vm method

### PR DESCRIPTION
**Pull request by wen-quan**
support more properties in create vm method , and according the max’s
suggestion , do some change code , and re-submit the change set.

### CODE REVIEW

File: softlayer/vm/softlayer_creator.go [hunk 1]
```diff
@@ -41,9 +41,14 @@ func NewSoftLayerCreator(softLayerClient sl.Client, agentEnvServiceFactory Agent
 	}
 }
 
+func (c SoftLayerCreator) getTimeStample(now time.Time) string {
+	//utilize the constants list in the http://golang.org/src/time/format.go file to get the expect time formats
```
Comment (maximilien) [inline softlayer/vm/softlayer_creator.go]: nice comment, thanks :smile:

File: softlayer/vm/softlayer_creator.go [hunk 2]
```diff
@@ -56,11 +61,19 @@ func (c SoftLayerCreator) Create(agentID string, stemcell bslcstem.Stemcell, clo
 			GlobalIdentifier: stemcell.Uuid(),
 		},
 
-		SshKeys:           cloudProps.SshKeys,
-		HourlyBillingFlag: true,
+		SshKeys: cloudProps.SshKeys,
+
+		HourlyBillingFlag: cloudProps.HourlyBillingFlag,
+		//Needed for ephemeral disk
```
Comment (maximilien) [inline softlayer/vm/softlayer_creator.go]: Cool, like this

**Comment by maximilien** [comment]
Hi, @wen-quan,

This looks good. Merging. 

Cheers :beers: 

@maximilien

PS: There is a `TODO` in the code this is not from you but before we started using Tracker. Can you please make sure in the Tracker we have a story for it or if the device name is correct and always the same for SL then we can remove the TODO. Thanks again.

**Comment by wen-quan** [comment]
@maximilien 
Great thanks for your help !

**Pull request merged**